torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -25,11 +25,10 @@
 #include <ATen/TensorSubclassLikeUtils.h>
 #include <c10/util/SmallBuffer.h>
 
-
-#include <ciso646>
 #include <algorithm>
-#include <numeric>
+#include <ciso646>
```

Comment:
Huh?

Reply:
hmm idk I think vscode did some trickery when formatting

Reply:
I would really much advise against autoformatting!
It adds a lot of noise to PRs and can do crazy stuff with import/include.

Reply:
yeah lucky it looks like all it did was rearrange the includes here but I deff understand the sentiment